server/services/group.js

```diff
@@ -14,7 +15,8 @@ function deleteGroup(groupName) {
 function createGroup(groupObject) {
   const groupSchema = Joi.object().keys({
     name: Joi.string().required(),
-    users: Joi.array().items(Joi.string().hex().length(24))
+    users: Joi.array().items(Joi.string().hex().length(24)),
+    escalationPolicy: Joi.object()
```

Comment:
I think this should be changed. The EP should be automatically created with the group and updated after creation if needed. 

I will do this in a separate PR so this doesn't get too cluttered. See https://github.com/PagerNation/PagerNation/issues/118